contracts/context-config/tests/sandbox.rs

```diff
@@ -1066,7 +1066,14 @@ async fn test_deploy() -> eyre::Result<()> {
     let success_value = res.raw_bytes()?;
     let proposal_result: serde_json::Value = serde_json::from_slice(&success_value)?;
     assert_eq!(proposal_result["num_approvals"], 1);
-    let created_proposal_id = proposal_result["proposal_id"].as_u64().unwrap();
+    let created_proposal_id: [u8; 32] = proposal_result["proposal_id"]
+        .as_array()
+        .expect("proposal_id should be a byte array")
+        .iter()
+        .map(|v| v.as_u64().unwrap() as u8)
+        .collect::<Vec<u8>>()
+        .try_into()
+        .expect("proposal_id should be 32 bytes");
```

Comment:
`proposal_id` really should've been represented as a new type`ProposalId(Identity)`, like the others in `config/types`

then we'd have been able to use `Repr<ProposalId>`

but for now, here, I think leaning in to `serde_json` should suffice

```suggestion
    let created_proposal_id: [u8; 32] = serde_json::from_value(proposal_result["proposal_id"].clone())?;
```

or alternatively, and perhaps preferably, deserialize to the return type `ProposalWithApprovals`